### ISSUE TITLE
Generate code for body-file.json

### ISSUE THREAD

**Issue by jianghaolu** [Azure/autorest.java#416]

**Comment by weidongxu-microsoft** [comment]
Reponse is actually a BinaryResponse, not SchemaResponse.

```
    - binary: true
      language:
        default:
          name: ''
          description: ''
      protocol:
        http:
          knownMediaType: binary
          mediaTypes:
          - image/png
          statusCodes:
          - '200'
```

**Comment by jianghaolu** [comment]
Closed via #436.
